gaj23/bigbirds
# Gaj/documentation/readme update 03 08

#### Purpose: 
> Check all that apply.

### What does this PR do? 
> Clearly and concisely describe changes made. Include screenshots as needed.

This PR includes a bug fix surrounding generating a random bird of the day. Additionally, further testing was applied. 

This PR also meets MVP as originally dictated.

Closes Issues #10, #13

### Problems Encountered & Resulting Solutions  
> State any problems encountered & include screenshots as needed. 
> Describe your code changes in detail for reviewers.

Major problems were encountered with my original attempts at having a bird randomly generate based on whether or not it exists on a user's seen list. I pivoted due to time constraints to only rendering an image of a bird. 

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

Please see `App` and 'Highlight` for a majority of the changes maded.

### How should this be tested?
> What steps need to be taken to confirm functionality? 

Please clone down locally and run `npm start`.

Use Cypress for testing
 - check that the following key-value pair in `scripts` is in your `package.json`
 - `"cypress": "cypress open"`
 - if not please download Cypress with `npm i -D cypress` and add the above to `scripts`.
 - type `npm run cypress` into CLI to run various test files


### Steps To Take
> Any questions and possible steps moving forward.

There are further considerations that need to be applied for future iterations. 

#### Checklist:
